### ISSUE TITLE
Should not show error message for failing to parse VERSION_CODE

### ISSUE THREAD

**Issue by hideki** [couchbase/couchbase-lite-java-core#434]
https://forums.couchbase.com/t/couchbase-lite-framework-version/2944

Customer see error message: 
`02-13 17:32:55.450 1620-1620/couchbase.examples.com.couchbasemobile E/CBLite﹕ Cannot parse version code:`

Current code: We display error message if failed to parse VERSION_CODE
```
private static final String VERSION_CODE="${VERSION_CODE}";  // replaced during build process
...
...
try {
   Integer.parseInt(VERSION_CODE);
} catch (NumberFormatException e) {
   Log.e(Log.TAG, "Cannot parse version code: %s", VERSION_CODE);
}
```

According to build engineer:
VERSION_CODE is not set for release build

Code changes following way:
In case VERSION_CODE is null or empty, set 0.
Also, if failed to parse VERSION_CODE, set 0

**Comment by pasin** [comment]
Does this issue affect 1.0.4 in Jenkin build?

**Comment by hideki** [comment]
Current CBL Android/Java v1.0.4 shows unnecessary error message because of  this.

**Comment by pasin** [comment]
I think you should cherry-pick this fix to 1.0.4 branch and let jenkin rebuild the 1.0.4 binary.

**Comment by tleyden** [comment]
+1
